workflows/proteomics/clinicalmp/clinicalmp-database-generation/iwc-clinicalmp-database-generation.ga

```diff
@@ -0,0 +1,348 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "The workflow begins with the Database Generation process. The Galaxy-P team has developed a workflow that collects protein sequences from known disease-causing microorganisms to build a comprehensive database. This extensive database is then refined into a smaller, more relevant dataset using the Metanovo tool.",
+    "comments": [],
+    "creator": [
+        {
+            "class": "Organization",
+            "identifier": "0000-0001-9818-0537",
+            "name": "Subina Mehta"
+        }
+    ],
+    "format-version": "0.1",
+    "license": "CC-BY-4.0",
+    "name": "Generate a Clinical Metaproteomics Database",
+    "release": "0.1",
+    "report": {
+        "markdown": "\n# Workflow Execution Report\n\n## Workflow Inputs\n```galaxy\ninvocation_inputs()\n```\n\n## Workflow Outputs\n```galaxy\ninvocation_outputs()\n```\n\n## Workflow\n```galaxy\nworkflow_display()\n```\n"
+    },
+    "steps": {
+        "0": {
+            "annotation": "Human SwissProt Protein Database",
+            "content_id": null,
+            "errors": null,
+            "id": 0,
+            "input_connections": {},
+            "inputs": [
+                {
+                    "description": "Human SwissProt Protein Database",
+                    "name": "Human SwissProt Protein Database"
+                }
+            ],
+            "label": "Human SwissProt Protein Database",
+            "name": "Input dataset",
+            "outputs": [],
+            "position": {
+                "left": 0,
+                "top": 207.8446261703461
+            },
+            "tool_id": null,
+            "tool_state": "{\"optional\": false, \"tag\": null}",
+            "tool_version": null,
+            "type": "data_input",
+            "uuid": "bb037df4-ef56-42ad-8b3e-93425bc7bdfa",
+            "when": null,
+            "workflow_outputs": []
+        },
+        "1": {
+            "annotation": "Tandem Mass Spectrometry (MS/MS) datasets",
```

Comment:
What are the requirements for these files ? How does a user know what is an appropriate input here ? Are those from some kind of reference or disease sample ? Generally speaking you don't need to add an annotation if you're just repeating the label, but if it's not obvious what one might want to use as input the annotation would be the right field to explain this.

Reply:
It basically is refering to the MSMS or proteomics dataset need to run the workflow

Reply:
Thank you for fixing my issues.